examples/go/README.md

```diff
@@ -0,0 +1,26 @@
+# http2preload demo
+
+A sample app demonstrating the usage of http2preload package.
+
+To run the sample, you'll need Go and App Engine SDK for Go installed.
```

Comment:
like, do you want people to just check out this repo and read these steps, or should we provide a runnable example?